webapp/src/workspace.ts

```diff
@@ -79,13 +80,30 @@ export function setupWorkspace(id: string) {
         case "idb":
             impl = indexedDBWorkspace.provider;
             break;
+        case "cloud":
+            impl = cloudWorkspace.provider;
```

Comment:
Just thinking aloud: It seems like the single `impl` workspace won't be sufficient to support blended workspaces (local, cloud, github).

Reply:
Yes, definitely blending is essential to the plan.